tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,292 @@
+import os
+import pytest
+import tempfile
+import shutil
+from tests.acceptance import (
+    run_gitlabform,
+)
+
+
+class TestGroupAvatar:
+    def setup_method(self):
+        # Use gitlabform logo
+        self.project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+        self.test_image_path = os.path.join(self.project_root, "docs/images/gitlabform-logo.png")
```

Comment:
Minor comment. For clear readability of the test codes, can we rename the variable? Maybe something like `test_image_path_absolute`?